lib/client-assets.php

```diff
@@ -619,3 +619,23 @@ function gutenberg_extend_block_editor_styles( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_extend_block_editor_styles' );
+
+/**
+ * Load the default editor styles.
+ * These styles are used if the "no theme styles" options is triggered.
+ *
+ * @param array $settings Default editor settings.
+ *
+ * @return array Filtered editor settings.
+ */
+function gutenberg_extend_block_editor_settings_with_default_editor_styles( $settings ) {
+	$editor_styles_file              = gutenberg_dir_path() . 'build/editor/editor-styles.css';
+	$settings['defaultEditorStyles'] = array(
+		array(
+			'css' => file_get_contents( $editor_styles_file ),
```

Comment:
Did we previously not use these styles?

Reply:
We don't have a way to get just these styles, previously we just received an array containing the editor styles the theme wants, it can or can't include these.

Reply:
This file is already loaded that way in Core and in other places in Gutenberg, so I'm not sure we need special treatment here.